TeamCode/src/main/java/org/firstinspires/ftc/teamcode/RobotMovementFieldCentric.java

```diff
@@ -54,7 +54,7 @@ public void runOpMode() {
             double rotX = x * Math.cos(-botHeading) - y * Math.sin(-botHeading);
             double rotY = x * Math.sin(-botHeading) + y * Math.cos(-botHeading);
 
-            rotX = rotX * 1.1;  // Counteract imperfect strafing
+            rotX = rotX * 1.4;  // Counteract imperfect strafing
```

Comment:
2 comments:

1. Consider making this a variable and allow the variable to be changed via buttons (maybe A/B, etc) so the driver can change the value while the program is running and tune the setting. This means you would not be redownloading the program everytime you change / guess this value. Once the driver has done some trial and error, you can then preset the value to whatever they have found to be the best. Note: you'll need to output the variable value via telemetry
2. Any thoughts on why we have imperfect strafing? Is it due to weight distribution? Something else? We should try to run some experiments at some point to identify the source of this problem.